package.json

```diff
@@ -48,9 +48,12 @@
     "ea-command-ci": "./build.js -run --settings ./settings-test.json --postgresUrl $PG_URL --command",
     "ea-start": "./build.js -run -watch --lint --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt",
     "ea-start-xpost-dev": "PORT=4000 ./build.js -run -watch --lint --settings ../ForumCredentials/settings-xpost.json --postgresUrlFile ../ForumCredentials/xpost-pg-conn.txt",
-    "playwright-db": "docker run --rm --name forum-magnum-test-postgres -e POSTGRES_PASSWORD=password -p 5433:5432 -v $PWD/schema/accepted_schema.sql:/docker-entrypoint-initdb.d/00_accepted_schema.sql ankane/pgvector",
+    "playwright-db": "docker run --rm --name forum-magnum-test-postgres -e POSTGRES_PASSWORD=password -p 5433:5432 -v $PWD/schema/accepted_schema.sql:/docker-entrypoint-initdb.d/00_accepted_schema.sql -v $PWD/playwright/fixtures.sql:/docker-entrypoint-initdb.d/01_fixtures.sql ankane/pgvector",
+    "playwright-db-crosspost": "docker run --rm --name forum-magnum-test-postgres-crosspost -e POSTGRES_PASSWORD=password -p 5434:5432 -v $PWD/schema/accepted_schema.sql:/docker-entrypoint-initdb.d/00_accepted_schema.sql -v $PWD/playwright/fixtures.sql:/docker-entrypoint-initdb.d/01_fixtures.sql ankane/pgvector",
     "playwright-test": "playwright test",
+    "playwright-test-crosspost": "CROSSPOST_TEST=true playwright test",
```

Comment:
Same for running locally, I would rather not add another type of test to keep things simple, although I appreciate that the two databases is a meaningful difference. I don't think it would be so bad to run two databases for the other tests though